chart/kubeapps/README.md

```diff
@@ -1,3 +1,4 @@
+<!--- app-name: Kubeapps -->
```

Comment:
We have to remove this change, otherwise, we got an error in Netlify 😅 

````
 Error: Error building site: "/opt/build/repo/chart/kubeapps/README.md:2:1": starting HTML comment with no end
12:03:08 PM: Total in 23 ms
````

Reply:
Ok, removed in #5288